jobs/create-initial-pr-comment.js

```diff
@@ -44,7 +48,10 @@ module.exports = async function (
     number: prDoc.number
   }))
 
-  if (issue.state !== 'open' || issue.locked) return
+  if (issue.state !== 'open' || issue.locked) {
```

Comment:
not sure if necessary, but maybe we want split the `||` out into two `if` statements, so we can log precisely why it doesn’t work.